src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -360,6 +360,20 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>LatestVersionPlayed__c</fullName>
+        <description>The last Elevate event version played resulting in an update to this Recurring Donation.</description>
```

Comment:
I noticed that the description/helptext on the `LastElevateEventPlayed__c` field is really really close to this new field. Only the word "event" makes them different. I think it might avoid confusion (for us in the future as well as for customers) if there was a bit more in the description/helptext on these two fields to further clarify their usage and especially their differences.
CC @mattwashington